bool hecu_unidriver_got_msg_from_hecu
bool hecu_alive
bool lf_tool_home
bool filter_tool_home
